### ISSUE TITLE
Tesseract integration for more languages

### ISSUE THREAD

**Issue by pramjana** [amebalabs/TRex#33]
Hi,

Hope you are doing well!

Is there a way you can harness the `tesseract` (which is already installed on my system - it just needs to run using a shortcut or be available via the menubar like your app) command line application, which can provide OCR output for many other languages?

Thank you for your consideration!

Best,
Pramod

**Comment by pramjan** [comment]
Hmm, I didn't realize it would be a lot of work. I was thinking maybe run the tesseract command with the captured image (in the background), and display the result in your app like usual.
I'm from India, and looking for OCR for Telugu, Hindi (this is just a parameter to pass to `tesseract`)

**Comment by melonamin** [comment]
Yeah, I understand, but I want TRex to be easy to use and work out of the box. I can't ask user to install additional tool on aside.

So to do it properly I need to take tessaract library and properly integrate it. Let me think about it a bit, there are a lot of potential users in India 😄

**Comment by melonamin** [comment]
@pramjana well, this took a bit longer than I wanted... 😅

Please try the [beta build](https://github.com/amebalabs/TRex/releases/tag/v1.9.0-BETA-1)
